### PR TITLE
Standalone prometheus scraper docker image

### PR DESCRIPTION
**Pull request by ziegfried**

### CODE REVIEW

File: docker/prometheus-scraper-standalone/src/env.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        case 'off':
+        case 'no':
+            return false;
+        default:
+            throw new ConfigError(`Invalid value ${JSON.stringify(val)} for boolean environment variable ${variable}`);
+    }
+};
+
+export const envInt = (variable: string, defaultValue?: number): number => {
+    return defaultValue!;
```
Comment (ccordi) [inline docker/prometheus-scraper-standalone/src/env.ts:50]:
```suggestion
    const val = parseInt(process.env[variable] ?? '');

    if (val == null || Number.isNaN(val)) {
        if (defaultValue != null) {
            return defaultValue;
        }
        throw new ConfigError(`Missing required environment variable ${variable}`);
    }
    return val;
```

Comment (ziegfried) [reply docker/prometheus-scraper-standalone/src/env.ts]: Oh wow, thanks for catching that!

File: docker/prometheus-scraper-standalone/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Prometheus Scraper
+
+A simple docker images that uses the `@splunkdlt/prometheus-scraper` package to periodically scrape metrics from prometheus endpoint and forward it to a Splunk HTTP Event Collector.
```
Comment (ccordi) [inline docker/prometheus-scraper-standalone/README.md:3]:
```suggestion
A simple docker image that uses the `@splunkdlt/prometheus-scraper` package to periodically scrape metrics from a prometheus endpoint and forward it to a Splunk HTTP Event Collector.
```

File: README.md [hunk 3]
```diff
@@ -40,6 +40,16 @@ A helper to collect stats (internal metrics) of a node process to send to Splunk
 
 <!-- PACKAGE-LIST-END -->
 
+## Docker Images
+
+<!-- DOCKER-IMAGE-LIST -->
+
+### [prometheus-scraper](./docker/prometheus-scraper-standalone)
+
+A simple docker images that uses the `@splunkdlt/prometheus-scraper` package to periodically scrape metrics from prometheus endpoint and forward it to a Splunk HTTP Event Collector.
```
Comment (ccordi) [inline README.md:49]:
```suggestion
A simple docker image that uses the `@splunkdlt/prometheus-scraper` package to periodically scrape metrics from a prometheus endpoint and forward it to a Splunk HTTP Event Collector.
```

**Review by ccordi — APPROVED** [review]
LGTM

**Pull request merged**